minoc/README.md

```diff
@@ -157,3 +157,16 @@ done
 unset TAGS
 docker image list minoc
 ```
+
```

Comment:
The main README primarily serves as docs for the config so I'd like to avoid adding helpful things like this usage example... they also introduce a bunch of things that need explaining (cert support in front end proxy, permissions related config of minoc, potentially that identity belonging to a group with write permission to that namespace).

please remove; I will probably even remove the bit about tagging because it is not actually useful/interesting to developers or deployers (just publishers, and it isn't complete for them).